### PR TITLE
Prevent canceled invoices from marked as paid

### PR DESCRIPTION
**Pull request by elias-summermatter**

### CODE REVIEW

**Review by michael-scheurer — CHANGES_REQUESTED** [review]
There are a lot of changes, where it seems they are coming from an internal IDE linting rule. We should not add that in this pr. It's hardly reviewable.

Also, I cannot see any change in the api, which would solve "If a new subscription is created, then deactivated, reactivated and then paid there is a mess with periods". How come?

File: packages/editor/src/client/atoms/formControlUrl.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import React, {useState, useCallback} from 'react'
-import {Form, Message} from 'rsuite'
+import React, {useCallback, useState} from 'react'
```
[the remark this answers is not in the document]
Comment (elias-summermatter) [reply packages/editor/src/client/atoms/formControlUrl.tsx:1]: Sure, the fix is that we prevent user from mark canceled invoices as paid.

[the remark this answers is not in the document]
Comment (elias-summermatter) [reply packages/editor/src/client/atoms/formControlUrl.tsx:1]: This is just the normal linting of the project which nobody did. And if its ok i will not revert this. :-)

[the remark this answers is not in the document]
Comment (michael-scheurer) [reply packages/editor/src/client/atoms/formControlUrl.tsx:1]: I think the linting is always done pre-commit via husky. See package.json in the root folder.

"husky": {
    "hooks": {
      "pre-commit": "pretty-quick --staged --pattern \"{examples,packages,services}/**/*.{js,ts,tsx,json}\""
    }
  },

**Comment by elias-summermatter** [comment]
@michael-scheurer @Itrulia  please review that i can merge it an close this issue today.

**Review by Itrulia — APPROVED** [review]

**Review by michael-scheurer — APPROVED** [review]

**Pull request merged**